core/src/com/unciv/ui/components/tilegroups/CityButton.kt

```diff
@@ -531,7 +531,8 @@ class CityButton(val city: City, private val tileGroup: TileGroup) : Table(BaseS
             // second tap on the button will go to the city screen
             // if this city belongs to you and you are not iterating though the air units
             if (DebugUtils.VISIBLE_MAP || viewingPlayer.isSpectator()
-                || (belongsToViewingCiv() && !tileGroup.tile.airUnits.contains(unitTable.selectedUnit))) {
+                || (belongsToViewingCiv() && !tileGroup.tile.airUnits.contains(unitTable.selectedUnit))
+                || (city.civ.gameInfo.isEspionageEnabled() && viewingPlayer.espionageManager.getSpyAssignedToCity(city)?.isSetUp() == true)) {
```

Comment:
Remove extraneous parentheses

Reply:
Interesting how you prefer relying on the && having a higher precedence over using parenthesis.

Reply:
More like, the code shows that it's separated into OR lines, so you can intiut that each OR line functions as expected...?
YMMV, I personally find it more readable without, but that may be an acquired taste...